### DISCUSSION TITLE
Random bullets in ToC

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#274] [Get Help] [answered]
Hello again - I noticed that my Table of Contents has bullets placed between sections:

![Screen Shot 2022-03-24 at 12 17 08 PM](https://user-images.githubusercontent.com/74433007/159974076-9b93b9d1-a880-4d17-9d5e-1fd74bfeb9ba.png)

It seems like they appear in between sections _after_ the section I'm currently viewing. In the screenshot, I am on the first page of the Introduction section.

The bullets seem awkward and misplaced. How can I make them go away?

**Comment by geealbers** [comment, marked as answer]
@mmpadilla15 I don't think I've seen that before. It feels like something about the structure of your project must not be what Quire is expecting it to be when it's building the content list. Here are some initial questions I have. And if you're able to ZIP up your content folder only (not the whole project) and attach it here, I can take a look.

1. Do the bullets change positions depending on what page you're on? 
2. If they don't change positions are they all above sub-sections like they seem to be in your screenshot?
3. Other than the bullets does the contents list appear correct? Like with the correct nesting? For instance, is the repetition of "I. Essay 1" intentional? In your markdown files do you have a section landing page with that title and then three other pages inside the same folder with that same title?

**Reply by mmpadilla15** [reply]
[content.zip](https://github.com/thegetty/quire/files/8364398/content.zip)

Hi @geealbers  - A zip file of my content folder is attached. To answer your other questions:

1. Yes. If I'm on the cover page, there are no bullets. If I'm on page 1 of Essay one, they are there, but not if I'm on page 2 or 3 of essay one. Does that make sense?
2. They do change positions and are in between each section, but not between the subsections.
3. Yes, the ToC is correct with the correct nesting. Although as I now look at it, it seems the nesting goes weird when I'm on one of the nested pages. In the screenshot below I'm on Intro 3 which is nested under Introduction, but it is now showing everything else nested under Introduction also.

![Screen Shot 2022-03-28 at 11 02 00 AM](https://user-images.githubusercontent.com/74433007/160439806-26264af4-63eb-4bfe-9169-ba266976263c.png)

Perhaps I've got my nesting incorrect in my files?

**Reply by geealbers** [reply]
In your themes/default/layouts/partials/contents-list/contents-list.html file, on line 115, can you try replacing this:

```
{{- if and $pageWithinThisScope (ne $contentsPageID .File.UniqueID) -}}
```

with this:

```
{{- if and $pageWithinThisScope (or (eq $contentsLocation ".Params.menu") (and (eq $contentsLocation ".Params.toc") (ne $contentsPageID .File.UniqueID))) -}}
```

**Comment by mmpadilla15** [comment]
@geealbers It worked! No more bullets! Thank you!

![Screen Shot 2022-03-28 at 11 55 02 AM](https://user-images.githubusercontent.com/74433007/160448744-7d8253c3-c22d-4264-a75f-906cbe7e479d.png)
